packages/webdriver/tests/request.test.ts

```diff
@@ -399,24 +400,44 @@ describe('webdriver request', () => {
             expect(result.message).toBe('Command can only be called to a Selenium Hub')
         })
 
-        it('should throw if timeout happens too often', async () => {
-            const retryCnt = 3
-            const req = new WebDriverRequest('POST', '/timeout', {}, true)
-            const reqRetryCnt = jest.fn()
-            req.on('retry', reqRetryCnt)
-            const result = await req.makeRequest({
-                protocol: 'https',
-                hostname: 'localhost',
-                port: 4445,
-                path: '/timeout',
-                connectionRetryCount: retryCnt,
-                logLevel: 'warn'
-            }, 'foobar').then(
-                (res) => res,
-                (e) => e
-            )
-            expect(result.code).toBe('ETIMEDOUT')
-            expect(reqRetryCnt).toBeCalledTimes(retryCnt)
+        describe('"ETIMEDOUT" error', () => {
+            it('should throw if timeout happens too often', async () => {
+                const retryCnt = 3
+                const req = new WebDriverRequest('POST', '/timeout', {}, true)
+                const reqRetryCnt = jest.fn()
+                req.on('retry', reqRetryCnt)
+                const result = await req.makeRequest({
+                    protocol: 'https',
+                    hostname: 'localhost',
+                    port: 4445,
+                    path: '/',
+                    connectionRetryCount: retryCnt,
+                    logLevel: 'warn'
+                }, 'foobar').then(
+                    (res) => res,
+                    (e) => e
+                )
+                expect(result.code).toBe('ETIMEDOUT')
+                expect(reqRetryCnt).toBeCalledTimes(retryCnt)
+            })
+
+            it('should extend error with info about executing command', async () => {
+                const spy = jest.spyOn(utils, 'extendTimeoutErrByExecCmd')
+
+                const req = new WebDriverRequest('GET', '/timeout', {}, true)
+                const reqOpts = {
+                    protocol: 'https',
+                    hostname: 'localhost',
+                    port: 4445,
+                    path: '/',
+                } as Options.WebDriver
+                await req.makeRequest(reqOpts, 'foobar').then((res) => res, (e) => e)
```

Comment:
There is nothing we do with the result here, so:
```suggestion
                await req.makeRequest(reqOpts, 'foobar')
```

Reply:
I make request to `/timeout` route which is mocked and always return timeout error. I can rewrite to `try catch` but in adjacent tests used solution with `then`.

Reply:
Maybe then something like this?
```suggestion
                await req.makeRequest(reqOpts, 'foobar')
                    // ignore error
                    .catch((e) => e)
```